MaxText/configs/base.yml

```diff
@@ -175,3 +175,6 @@ stack_trace_interval_seconds: 600  # Stack trace collection frequency in seconds
 
 # Use iota operator in Embed
 use_iota_embed: False
+
+#Monitoring parameters
```

Comment:
Please write a note explaining this a little more carefully. "Export in-workload metrics to Cloud monitoring"

Reply:
Also have you visualized these metrics? Where do I see them, etc? Probably when we print tensorboard stuff we should also print a link to see these metrics.

Reply:
The metrics can be visualized on our monitoring dashboard titled 'Maxtext Metrics'. I've modified to print a link that points to all the project's dashboards, where one can create a new dashboard to use these metrics or add to existing dashboards.

Reply:
Can you post a link for a run you did?

Reply:
https://pantheon.corp.google.com/monitoring/dashboards/builder/0392c0a4-317c-4385-b446-296784f7c97b;startTime=2023-11-30T08:16:10.096Z;endTime=2023-12-03T02:05:25.320Z?e=13802955&mods=allow_workbench_image_override&project=tpu-prod-env-multipod